sound/soc/sof/Makefile

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
 
 snd-sof-objs := core.o ops.o loader.o ipc.o pcm.o pm.o debug.o topology.o\
-		control.o trace.o iomem-utils.o sof-audio.o
+		control.o trace.o iomem-utils.o sof-audio.o sof-utils.o
```

Comment:
didn't we update sof-test already? can this be dropped?

Reply:
We did, but after I have pushed this version.

@lyakh suggested that we should not add a new module and just compile the two split objects into `snd_sof`
What do you think?